src/components/ImportForm/PipelineSection/__tests__/PipelineSection.spec.tsx

```diff
@@ -41,4 +41,30 @@ describe('PipelineSection', () => {
     expect(screen.getByText(exampleDescription)).toBeInTheDocument();
     expect(screen.getByText(examplePipeline)).toBeInTheDocument();
   });
+
+  it('should display pipeline detail when a pipeline with detail is selected', () => {
+    formikRenderer(<PipelineSection />, { pipeline: 'fbc-builder' });
+    const expectedDetail = 'The File-Based Catalog (FBC) builder pipeline creates container images for operator bundles using the file-based catalog format. This pipeline supports building OLM operator catalogs with improved performance and maintainability compared to the deprecated SQLite-based format.';
```

Comment:
is it possible to read it from variables like BuildPipelineConfigJson then the code would be more clean?

Reply:
I updated these.